### PR TITLE
[SYCLomatic] Add a dummy function to migrate API like getErrorStr/getErrorName

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Comment by zhiweij1** [comment]
SYCLCompat API added https://github.com/intel/llvm/pull/15719

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/test/dpct/cuda-get-error-string.cu [hunk 1]
```diff
@@ -2,64 +2,49 @@
 // RUN: FileCheck %s --match-full-lines --input-file %T/cuda-get-error-string/cuda-get-error-string.dp.cpp
 // RUN: %if build_lit %{icpx -c -fsycl %T/cuda-get-error-string/cuda-get-error-string.dp.cpp -o %T/cuda-get-error-string/cuda-get-error-string.dp.o %}
 
+#include "cublas_v2.h"
+#include "cusparse_v2.h"
+#include "cuda.h"
+#include "cudnn.h"
+#include "nccl.h"
```
Comment (zhiweij1) [inline clang/test/dpct/cuda-get-error-string.cu]:
```suggestion
#include "cuda.h"
```

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**